modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-fragment/itemActions.spec.ts

```diff
@@ -325,6 +320,138 @@ test.describe('Item Actions in Data Set fragment', () => {
 		});
 	});
 
+	test('Link item action works in Cards, List and Visualization modes', async ({
+		dataSetFragmentPage,
+		dataSetManagerApiHelpers,
+		layout,
+		page,
+	}) => {
+		await test.step('Create sample data for the data set cards and list', async () => {
+			await dataSetManagerApiHelpers.createDataSetCardsSection({
+				dataSetERC,
+				fieldName: 'id',
+				name: 'title',
+			});
+
+			await dataSetManagerApiHelpers.createDataSetListSection({
+				dataSetERC,
+				fieldName: 'id',
+				name: 'title',
+			});
+		});
+
+		await test.step('Create a link Item Action with an interpolated argument', async () => {
+			await dataSetManagerApiHelpers.createDataSetItemAction({
+				dataSetERC,
+				label_i18n: {en_US: LINK_ITEM_ACTION_NAME},
+				target: EItemActionTarget.LINK,
+				url: '/detail/{id}',
```

Comment:
To ease testing, I've created a link action with a parameter and then check that the parameter is interpolated correctly.
This way, I try to prevent multiple navigations back and forth after running an action.